### ISSUE TITLE
Repeating output

### ISSUE THREAD

**Issue by qwertystars** [OpenInterpreter/open-interpreter#1127]
### Describe the bug

When rendering Markdown content, particularly within code snippets or text areas, the text is being repeated multiple times, despite it being intended to display only once. This repetition occurs within the rendered output, making it difficult to read and understand the Markdown content properly.
This maybe a issue of powershell. if Yes please create the gui fast.

### Reproduce

Write this as prompt or any prompt that will make it generate output longer than the display size of monitor use. Make me a simple Pomodoro app.

### Expected behavior

The Markdown content should be displayed exactly once, providing a clear and concise representation of the intended text.

### Screenshots

![Screenshot 2024-03-25 134756](https://github.com/OpenInterpreter/open-interpreter/assets/62981066/62515734-5102-4ec1-9c22-f0043b482a3f)

### Open Interpreter version

0.2.4

### Python version

3.10.7

### Operating System name and version

Windows 11

**Comment by rustom** [comment]
Reproduced on MacOS, I don't think it's a Powershell issue. +1

**Comment by Notnaton** [comment]
This is a known issue. 
We rerender the output of the llm each time we get a token. This causes it to print a new "screen". 
Moving to textual would solve this, I have explored doing that. Just need some motivation 😅

**Comment by bars0um** [comment]
Not sure if this is an ok fix, but it appears that if I switch the Live attribute vertical_overflow to "elipses" on line [12](https://github.com/OpenInterpreter/open-interpreter/blob/3cffb8eaecb1009069183835e76dfd039a64438b/interpreter/terminal_interface/components/base_block.py#L12) of [terminal_interface/components/base_block.py](https://github.com/OpenInterpreter/open-interpreter/blob/main/interpreter/terminal_interface/components/base_block.py) the issue somewhat subsides...I've not tested this fully yet and it may end up causing a different issue if your file output is bigger than your console height...

**Comment by davidkan6666** [comment]
Have the similar issue, the output will be repeated many times:
![image](https://github.com/user-attachments/assets/d29d1ad9-79cd-47a9-a003-bce5f6e184cf)

**Comment by dandarm** [comment]
I'm experiencing the same problem still today, using gpt-4o-mini. 
Any solution up to now?

**Comment by interpreterwork** [comment]
Thank you for reporting this and for helping build the classic Python version of Open Interpreter. This issue concerns the earlier Python/LiteLLM/Jupyter product; this repository now tracks the Rust Open Interpreter built on Codex. We are closing the legacy report here so current product tracking starts clean. If you still want to pursue it, the community-maintained classic fork is https://github.com/endolith/open-interpreter. If the same problem affects the current Interpreter release, please open a new issue with the current template and include the version, provider, model, wire API, harness, platform, and reproduction.
